ludwig/schema/combiners/utils.py

```diff
@@ -16,15 +19,17 @@ def get_combiner_jsonschema():
     """Returns a JSON schema structured to only require a `type` key and then conditionally apply a corresponding
     combiner's field constraints."""
     combiner_types = sorted(list(combiner_registry.keys()))
+    parameter_metadata = convert_metadata_to_json(COMBINER_METADATA[TYPE])
     return {
         "type": "object",
         "properties": {
             "type": {
                 "type": "string",
                 "enum": combiner_types,
                 "default": "concat",
-                "title": "type",
+                "title": "Combiner Type",
```

Comment:
Actually I think we should revert this change. Otherwise the combiner page will read Combiner as a super heading and then "Combiner Type" as a sub-heading. I think we'll do the prefixing on the frontend side (CC @martindavis ).

Also as a general note we should keep things on the Ludwig side in snake case IMO.

Reply:
Ah ok, yes this is indeed your wheelhouse so will change back!